### ISSUE TITLE
Invalid Username On Signup

### ISSUE THREAD

**Issue by dragazo** [NetsBlox/NetsBlox#3378]
Some students are having issues where they cannot create NetsBlox accounts for saving projects. When they put in their username, they get a generic invalid username error. One of the students tried username `yedina` and augmented versions like `yedina123`. Could be related to the profanity filter, though I'm not sure what the issue is for this name in particular.

**Comment by brollb** [comment]
Yeah, it looks like the profane checker is being a little overzealous and doesn't like "yed". If you use the CLI, you can bypass the profanity checker. Otherwise, the student may need to choose a different one for now :/

**Comment by brollb** [comment]
(I had to do some google "yed" to figure out why it might be an issue. It seems pretty obscure - or I am just getting old ;) )

**Comment by gsteinLTU** [comment]
Did you find anything more specific? I found a few things saying it's possibly offensive Thai slang, but I'm leaning more that it was a typo for "yid".

There seem to be a lot of valid words it blocks, like "japanese" or "titan" (which could be a popular username in Nashville). Looks like there's a package that tries to fix it: https://www.npmjs.com/package/bad-words-relaxed , but it's pretty old.
